hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/invalidation/MetaDataGenerator.java

```diff
@@ -37,17 +37,17 @@
  */
 public class MetaDataGenerator {
 
-    private final int partitionCount;
-    private final ConcurrentMap<String, AtomicLongArray> sequenceGenerators = new ConcurrentHashMap<String, AtomicLongArray>();
-    private final ConstructorFunction<String, AtomicLongArray> sequenceGeneratorConstructor
+    final int partitionCount;
+    final ConcurrentMap<String, AtomicLongArray> sequenceGenerators = new ConcurrentHashMap<String, AtomicLongArray>();
+    final ConstructorFunction<String, AtomicLongArray> sequenceGeneratorConstructor
             = new ConstructorFunction<String, AtomicLongArray>() {
         @Override
         public AtomicLongArray createNew(String arg) {
             return new AtomicLongArray(partitionCount);
         }
     };
-    private final ConcurrentMap<Integer, UUID> uuids = new ConcurrentHashMap<Integer, UUID>();
-    private final ConstructorFunction<Integer, UUID> uuidConstructor
+    final ConcurrentMap<Integer, UUID> uuids = new ConcurrentHashMap<Integer, UUID>();
+    final ConstructorFunction<Integer, UUID> uuidConstructor
```

Comment:
I see the need to make `sequenceGenerators` package-private, why also expose the rest?